gamification/process/activity.py

```diff
@@ -0,0 +1,30 @@
+from gamification.data.labels import used_labels
+
+
+def get_activity(activity_type, labels):
+    """
+    Get an activity based on the labels on issues or mrs.
+
+    :param activity_type: a string representing the type of the
+                          activity. e.g. issue or merge_request.
+    :param labels:        a QuerySet dict containing the 'name'
+                          as key and 'name of the label' as value.
+    :return:              a touple of points and activity string.
+    """
+    labels_list = [label['name'] for label in labels]
+
+    # Removing labels which are not in used_labels
+    for label in labels_list[:]:
+        if label not in used_labels():
+            labels_list.remove(label)
+
+    # Sorting the labels to get consistent activity
+    labels_list.sort()
+    if activity_type == 'issue':
+        activity = 'Created a '
+        activity = activity + ' '.join(labels_list) + ' issue'
+        return activity
+    elif activity_type == 'merge_request':
```

Comment:
can you see similarity between these two branches ?

Reply:
yes, similar case ad https://github.com/coala/community/pull/160#discussion_r208126754